packages/manager/src/pages/profile/settings/profile-image/profile-image-form.js

```diff
@@ -0,0 +1,104 @@
+import "./profile-image.scss";
+
+import { withFormik } from "formik";
+import { GVButton } from "gv-react-components";
+import React from "react";
+import { translate } from "react-i18next";
+import { compose } from "redux";
+import InputPhoto, {
+  getInputPhotoInitialValue
+} from "shared/components/form/input-photo/input-photo";
+import UserIcon from "shared/media/user-avatar.svg";
+
+import ProfileImageValidationSchema from "./profile-image.validators";
+
+class ProfileImageForm extends React.Component {
```

Comment:
Remove the differences between investor and manager versions and move to `shared/components`